inception-project/inception
# #649 - Learning curve diagram

**What's in the PR**
- Added Learning Curve Diagram 

**How to test manually**
* Add show.learning.curve.diagram=true in the settings.properties
* Create project
* Add document.
* Add recommender (must be enabled and no-evaluation unselected)
* Annotate the document on the layer recommender was added.
* Learning curve will appear on the recommender sidebar

**Automatic testing**
* [x] PR includes integration tests of the DAO functions.

**Documentation**

**Todo**
- [x] Need to fix checkstyle
- [x] Adhere to DkPro Code Style
- [x] Fix error when there are no evaluations
- [x] Add configuration setting to show chart (hidden by default)
- [x] Use recommender id to distinguish recommenders and show recommender name on the learning chart.
- [x] The specified number of points for each recommender must be plotted.
